packages/adapter-vercel/index.js

```diff
@@ -407,8 +407,22 @@ function static_vercel_config(builder) {
 		overrides[page.file] = { path: overrides_path };
 	}
 
+	/** @type {Record<string, any> | undefined} */
+	let images = undefined;
+	const img_config = builder.config.kit.images;
+	if (config.images || img_config.loader === '@sveltejs/adapter-vercel/image-loader') {
+		images = {
```

Comment:
Why not just let the user set these directly? I don't see what we gain by hiding it from them